packages/@ember/-internals/glimmer/lib/component-managers/outlet.ts

```diff
@@ -103,7 +103,10 @@ class OutletComponentManager
         let mountPoint = currentOwner.mountPoint;
 
         state.engine = currentOwner;
-        state.engineBucket = { mountPoint };
+
+        if (mountPoint) {
+          state.engineBucket = { mountPoint };
+        }
```

Comment:
The code that uses this seems to expect that `engineBucket` is only set if it has a `mountPoint`:

https://github.com/emberjs/ember.js/blob/649114f70f2c9288338845b50d7f18e519fc7c7f/packages/%40ember/-internals/glimmer/lib/component-managers/outlet.ts#L135-L144